include/solvers/gls_navier_stokes.h

```diff
@@ -120,19 +120,25 @@ class GLSNavierStokesSolver
   void
   setup_preconditioner();
 
-
   /**
-   * @brief  defined the non zero constraints used to solve the problem.
+   * @brief Define the non-zero constraints used to solve the problem.
    */
   void
   define_non_zero_constraints();
 
   /**
-   * @brief defined the zero_constraints used to solved the problem.
+   * @brief Define the zero constraints used to solved the problem.
    */
   void
   define_zero_constraints();
 
+  /**
+   * @brief Define the zero constraints used to solved the problem that change
+   * with physic solutions.
+   */
+  void
+  setup_dynamic_zero_constraints();
```

Comment:
Just for consistence with the two previous functions.

```suggestion
  define_dynamic_zero_constraints();
```